src/imitation/scripts/expert_demos.py

```diff
@@ -38,13 +44,20 @@ def rollouts_and_policy(
   rollout_save_n_timesteps: Optional[int] = None,
   rollout_save_n_episodes: Optional[int] = None,
 
+  ray_tune_interval: int = 1,
```

Comment:
Is this meant to be `1` not `-1` (usually we've kept the default value here and in the Sacred config the same)?

Reply:
(This will be removed because we are no longer using Ray Tune track or reporter)